frontend/app/dashboard/page.tsx

```diff
@@ -153,6 +155,23 @@ export default function Dashboard() {
           </div>
         </div>
       </div>
+      {eventDataList.length === 0 ? (
```

Comment:
you can use && instead of ? : (ternary) such that is {<condition> && <HTML ELEMENT>} and it will only display if condition is true

Reply:
ahhh yes good suggestion